plugins/aws/aws.plugin.zsh

```diff
@@ -152,7 +152,7 @@ compctl -K _aws_profiles asp acp aws_change_access_key
 # AWS prompt
 function aws_prompt_info() {
   [[ -z $AWS_PROFILE ]] && return
-  echo "${ZSH_THEME_AWS_PREFIX:=<aws:}${AWS_PROFILE}${ZSH_THEME_AWS_SUFFIX:=>}"
+  echo "${ZSH_THEME_AWS_PREFIX:=<aws}:${AWS_PROFILE}> <region:${AWS_REGION}${ZSH_THEME_AWS_SUFFIX:=>}"
```

Comment:
There seems to be a typo here, around `<aws:}$` - please check.

Reply:
oh, let me change it